profile/README.md

```diff
@@ -0,0 +1,64 @@
+# Infrastructure Made Easy
+
+As a group of engineers focusing on creating platforms for developers, we found ourselves
+consistently asking the same question: _how do we make the creation and management of infrastructure
+as simple as possible for our customers so that they can focus on writing code?_
+
+Simple question, not so simple to answer! When we first began talking amongst ourselves, we always
+hoped that someone would have a concrete answer, but the only thing we found was that we all had
+difficulty coming up with one. Our experience convinced us that this was a problem worth considering
+and exploring.
+
+This GitHub organization is our attempt to publicize our experiments and thoughts around this
+question. We have no answer, but we hope you'll follow along as we stumble through.
+
+## Tenets
+
+### Define Infrastructure as Code
+
+There is a reason that [the Infrastructure as Code
+movement](https://www.hashicorp.com/resources/what-is-infrastructure-as-code) has gained so much
+popularity in recent years. Ensuring that we define infrastructure in code has many advantages,
+such as:
+
+- It ensures consistency when we provision infrastructure across many environments and regions.
+- It allows the construction of reusable components that define small, single-use pieces of
+  infrastructure, which can then be composed into larger units, much like using libraries in
+  application code.
+- It enables infrastructure provisioning and ongoing management to be automated in the same way as
+  we build and release software.
+- It allows for versioning the definition of your infrastructure, potentially in
+  concert/coordination with the software running on the infrastructure.
+
+For these reasons, we reject any solution that does not allow us to generate, export, or write the
+definition of infrastructure in a form compatible with version-controlled storage.
+
+### Do Not Force Provider Choice
+
+We do not consider solutions that are inherently proprietary or useful only in the context of a
+single cloud provider since this limits their usefulness and leads to the necessity of learning
+multiple languages/frameworks.
```

Comment:
Would you consider Kubernetes a single provider?

Reply:
I would not because you can run Kubernetes on many different cloud platforms (AWS, GCP, Azure, on-prem, etc.). My comment here was geared more towards avoiding proprietary/single-provider solutions, like Azure ARM templates, various CDKs, or CloudFormation.